apps/frontend/src/pages/access-management/access-management.tsx

```diff
@@ -75,64 +86,60 @@ const AccessManagement = (): JSX.Element => {
 		onOpen: onDeleteModalOpen,
 	} = useModal();
 
-	const [selectedGroupId, setSelectedGroupId] = useState<null | number>(null);
-
-	const hasSelectedGroup = Boolean(selectedGroupId);
-
-	useEffect(() => {
+	const fetchUsers = useCallback(() => {
```

Comment:
handleLoadUsers / handleLoadGroups
```suggestion
	const handleLoadUsers = useCallback(() => {
```